contrib/julia-config.jl

```diff
@@ -51,6 +51,8 @@ end
 function ldlibs()
     if is_unix()
         return replace("""-Wl,-rpath,$(libDir()) -ljulia""","\\","\\\\")
+    elseif is_windows()
+        return replace("""-ljulia -lopenlibm""","\\","\\\\")
```

Comment:
we don't support anything that is neither unix nor windows. if we did, who know whether it would use backslashes in paths...